components/gcp-click-to-deploy/ks-app/components/params.libsonnet

```diff
@@ -3,7 +3,7 @@
   components: {
     ingress: {},
     webapp: {
-      image: "gcr.io/kubeflow-images-public/gcp-click-to-deploy:v20180720-5a864c2c-dirty-f1a621",
+      image: "gcr.io/kubeflow-images-public/gcp-click-to-deploy:v20190725-v0.4.0-rc.1-610-g41443e5e",
```

Comment:
I guess it's not been used anywhere?
Maybe remove configs here?

Reply:
I intend to.  but let me finish on another part and have another followup PR to remove it for safety.